### PR TITLE
EN-GB translation

### PR DESCRIPTION
**Pull request by expobrain**
Translated most of the website to English (british) language

### CODE REVIEW

File: _projects/en/anpr.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ---
 title: ANPR
 logo: /assets/images/logo-anpr@2x.png
-subtitle: Anagrafe Nazionale della Popolazione Residente
-payoff: L'anagrafe diventa unica e digitale
-description: Un'unica anagrafe centrale, raggiungibile anche via API, che mantiene le informazioni aggiornate su residenza, stato di famiglia, e molto altro.
+subtitle: National Registry of Resident Population
+payoff: The registry become one and digital
+description: >
+  A single centralised registry, accessible by an API, which holds the up-to-date informations
```
Comment (gvarisco) [inline _projects/en/anpr.md]: "informations" -> "information" (uncountable noun)

File: _projects/en/fatturapa.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 title: FatturaPA
-subtitle: La fattura elettronica
+subtitle: The electronic invoice
 logo: /assets/images/logo-fatturapa@2x.png
-payoff: La fatturazione elettronica verso lo Stato e tra privati
+payoff: The electronic invoice between the Governemnt and the private sector
 description: >
-  Un formato XML per descrivere in modo non ambiguo una fattura: scopri come può essere utile, anche tra privati, e come integrarlo nei tuoi
-  software gestionali.
+  An XML format to describe without ambiguity an invoce: discover how this can be useful, even
```
Comment (gvarisco) [inline _projects/en/fatturapa.md]: Typo: invoce -> invoice

File: _projects/en/anpr.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
 ---
 title: ANPR
 logo: /assets/images/logo-anpr@2x.png
-subtitle: Anagrafe Nazionale della Popolazione Residente
-payoff: L'anagrafe diventa unica e digitale
-description: Un'unica anagrafe centrale, raggiungibile anche via API, che mantiene le informazioni aggiornate su residenza, stato di famiglia, e molto altro.
+subtitle: National Registry of Resident Population
+payoff: The registry become one and digital
```
Comment (gvarisco) [inline _projects/en/anpr.md]: become -> becomes

File: _projects/en/anpr.md [hunk 4]
```diff
@@ -15,24 +17,36 @@ layout: project
...
+ANPR is a memorable project: instead of having more than 8,000 registries spread over the entire
+national territory (in evey municipality), Italy will have finally a single centralised registry
+which will simplify all the interactions with the Public Administration.
 
-Ma questo è solo l'inizio: avete mai pensato perché dovete spesso chiedere dei certificati all'anagrafe? Semplice: ad oggi non esiste un'anagrafe unica centralizzata a cui gli enti pubblici possono rivolgersi per accedere alle informazioni senza doverle chiedere ogni volta ai cittadini. Con ANPR, tutto cambia.
+In the future we hope to provide more features like update your residency's address or download a
+Family Status Declaration straight from the website.
 
-Ad oggi, l'accesso ad ANPR è riservato solo agli operatori anagrafici e ai software delle anagrafi comunali. Questa sezione è quindi dedicata a tutti coloro che stanno supportando i comuni nella fase di passaggio ad ANPR e ci stanno aiutando a realizzare questa visione. C'è ancora molto da fare e il contributo di tutti è importante. 
+This is only the beginnig: did you ever asked yourself why you need to request certificates at the
```
Comment (gvarisco) [inline _projects/en/anpr.md]: Typo: beginnig -> beginning

File: _projects/en/18app.md [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
+title: 18App and Carta Docente
+subtitle: Bonus culture and education
 logo: /assets/images/logo-18app-cartadocente@2x.png
-payoff: La piattaforma dove spendere i bonus cultura e formazione
+payoff: The platform where to spend the culture and education's bonus
 description: >
-  Se sei un esercente e vuoi integrarti con la piattaforma, qui puoi trovare le informazioni dettagliate per come integrarsi in modo efficace
-  con la piattaforma.
+  If you are a business owner and you want to integrate with the platform here you can find all the
+  detailed informations about how to integrate efficently with the platform
```
Comment (gvarisco) [inline _projects/en/18app.md]: "informations" -> "information" (uncountable noun)

File: _projects/en/anpr.md [hunk 6]
```diff
@@ -15,24 +17,36 @@ layout: project
...
+registry in your municipality? Simple: at today a single centralised registry where the public
+offices can access your informations doesn't exists at all. With ANPR it will completely different.
```
Comment (gvarisco) [inline _projects/en/anpr.md]: "informations" -> "information" (uncountable noun)

File: _faqs/en/1.yml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+---
+title: What's the goal of this website?
+text: >
+  The website was created to simplify and improve the interaction between the developers and the
+  Public Administration, starting from some important projects like the ANPR, the National Registry
+  of the Resident Population, or the SPID, the digital management system of the public identity.
+  Until now except some rare exceptions the central Public Administration limited itself on
+  drawfting laws and regulations in a non-technicsal bureaucratic language without any tool or
```
Comment (gvarisco) [inline _faqs/en/1.yml]: Typo: non-technicsal -> non-technical

File: _faqs/en/10.yml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+---
+title: Who is managing this website?
+text: >
+  **Developers Italia** is a website promoted by the
+  [Team di Trasformazione Digitale](https://teamdigitale.governo.it/) and it's managed in
```
Comment (gvarisco) [inline _faqs/en/10.yml]: Team di Trasformazione Digitale -> Digital Transformation Team

File: _faqs/en/20.yml [hunk 9]
```diff
@@ -0,0 +1,16 @@
+---
+title: What the mmeaning of "project's maintainer"?
```
Comment (gvarisco) [inline _faqs/en/20.yml]: Typo: mmeaning -> meaning

File: _includes/footer--simple.html [hunk 10]
```diff
@@ -11,7 +11,7 @@
     </div>
 
     <div class="Grid-cell u-md-size1of2 u-lg-size1of3 u-padding-r-bottom td-footer">
-      <p class="u-padding-bottom-l" style="opacity: 0.5">In collaborazione con</p>
+      <p class="u-padding-bottom-l" style="opacity: 0.5">{{ site.data.l10n.[active_lang].t.in_collaboration_with }}</p>
```
Comment (gvarisco) [inline _includes/footer--simple.html]: good idea!

File: _faqs/en/20.yml [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+text: >
+  In this website for "maintaner" we mean the people in charge of write the source code, the
```
Comment (gvarisco) [inline _faqs/en/20.yml]: Typo: of write -> of writing

File: _faqs/en/20.yml [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+  documentation, and the examples of the projects, also they are in charge of interact with the
```
Comment (gvarisco) [inline _faqs/en/20.yml]: Typo: of interact -> of interacting

File: _faqs/en/20.yml [hunk 13]
```diff
@@ -0,0 +1,16 @@
...
+  developers's community giving them support on adopting the technology.
```
Comment (gvarisco) [inline _faqs/en/20.yml]: Typo: developers's -> developers'

File: _faqs/en/20.yml [hunk 14]
```diff
@@ -0,0 +1,16 @@
...
+  Tipically the *maintaners* works in closely with the product owner of the project and with the
```
Comment (gvarisco) [inline _faqs/en/20.yml]: Typo: Tipically -> Typically

File: _faqs/en/20.yml [hunk 15]
```diff
@@ -0,0 +1,16 @@
...
+  technical staff of the Public Administration. For example: PagoPA, the centralised payment system
+  of the Public Administration, it's project of the AgID (Agenzia per l'Italia Digitale); the
```
Comment (gvarisco) [inline _faqs/en/20.yml]: Typo: it's project -> it's a project

File: _faqs/en/20.yml [hunk 16]
```diff
@@ -0,0 +1,16 @@
...
+  *maintainers* which you can see in Developers Italia will help AgID and will provide support to
+  the community to adopt PagoPA by thousand of people, building and maintaining the plugins for the
+  most common CMS, interface libraries for Python or Ruby, and documentation of the APIs in
```
Comment (gvarisco) [inline _faqs/en/20.yml]: Typo: CMS -> CMSes

File: _faqs/en/30.yml [hunk 17]
```diff
@@ -0,0 +1,14 @@
+---
+title: How do you choose a project's maintainer?
+text: >
+  As in every community build around an open source project the *maintainer* of the project are
```
Comment (gvarisco) [inline _faqs/en/30.yml]: Typo: are chosen -> is chosen

File: _faqs/en/30.yml [hunk 18]
```diff
@@ -0,0 +1,14 @@
...
+  chosen based on it technical expertise, knowledge of the project, the level of partecipation in
```
Comment (gvarisco) [inline _faqs/en/30.yml]: Typo: it -> its

File: _faqs/en/30.yml [hunk 19]
```diff
@@ -0,0 +1,14 @@
...
+  the community itself, and often in big projects they receive a compensation because their put
```
Comment (gvarisco) [inline _faqs/en/30.yml]: Typo: their -> they

File: _faqs/en/30.yml [hunk 20]
```diff
@@ -0,0 +1,14 @@
...
+  huge effor in terms of time and resources to contribute to the project itself.
```
Comment (gvarisco) [inline _faqs/en/30.yml]: Typo: effor -> effort

File: _faqs/en/30.yml [hunk 21]
```diff
@@ -0,0 +1,14 @@
...
+  A *maintainer* it's not only a supervisor but he must be also the first in terms of quantity and
```
Comment (gvarisco) [inline _faqs/en/30.yml]: Typo: it's -> is

File: _faqs/en/60.yml [hunk 22]
```diff
@@ -0,0 +1,11 @@
+---
+title: >
+  I've release an open source software which can be useful to the community. Can I add it to this
```
Comment (gvarisco) [inline _faqs/en/60.yml]: Typo: release -> released

File: _faqs/en/20.yml (hunk 11 above)
Comment (rasky) [inline _faqs/en/20.yml]: Also typo: maintaner -> maintainer

**Comment by expobrain** [comment]
Also I wasn't able to add the route `/en/news` as well because of my limited knowledge of Jekyll

**Comment by rasky** [comment]
perfect, I guess other fixes can just be followups

**Comment by rasky** [comment]
thanks @expobrain

**Pull request merged**